tests/test_api_requests.py

```diff
@@ -211,3 +212,27 @@ async def test_job_id(self, job_id):
             assert resp.status == 200
             assert "zyte-api" in resp.flags
             assert resp.body == f"<html>{job_id}</html>".encode("utf8")
+
+
+def test_api_key_presence():
+    from scrapy_zyte_api.handler import ScrapyZyteAPIDownloadHandler
+
+    API_KEY = "TEST_API_KEY"
+
+    # Setting the API KEY via env vars should work
+    os.environ["ZYTE_API_KEY"] = API_KEY
```

Comment:
a nitpick: it'd be good to restore os.environ to its previous state on the test teardown